### PR TITLE
Parking crud

### PR DESCRIPTION
**Pull request by maloun96**

### CODE REVIEW

File: app/Http/Controllers/Backend/ParkingController.php [hunk 1]
```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace App\Http\Controllers\Backend;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\Backend\Auth\User\StoreUserRequest;
```
Comment (volosinpaul) [inline app/Http/Controllers/Backend/ParkingController.php]: StoreUserRequest imported but not used

File: app/Http/Requests/Backend/Parking/StoreParkingRequest.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	 */
+	public function rules()
+	{
+		return [
+			'name'     => ['required', 'max:191', Rule::unique('parking')],
+			'status'     => ['boolean'],
+			'capacity'     => ['required', 'max:191', 'integer'],
+			'price'     => ['required', 'max:191', 'integer'],
+			'latitude'     => ['required', 'regex:/^[-]?(([0-8]?[0-9])\.(\d+))|(90(\.0+)?)$/'],
+			'longitude'     => ['required', 'regex:/^[-]?(([0-8]?[0-9])\.(\d+))|(90(\.0+)?)$/'],
```
Comment (volosinpaul) [inline app/Http/Requests/Backend/Parking/StoreParkingRequest.php]: 'The longitude format is invalid.' response on inputting valid value for longitude. The regex for longitude needs to be updated.
Latitudes range from -90 to 90.
Longitudes range from -180 to 180.

File: app/Http/Requests/Backend/Parking/UpdateParkingRequest.php [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+	 * @return array
+	 */
+	public function rules()
+	{
+		return [
+			'status'     => ['boolean'],
+			'capacity'     => ['required', 'max:191', 'integer'],
+			'price'     => ['required', 'max:191', 'integer'],
+			'latitude'     => ['required', 'regex:/^[-]?(([0-8]?[0-9])\.(\d+))|(90(\.0+)?)$/'],
+			'longitude'     => ['required', 'regex:/^[-]?(([0-8]?[0-9])\.(\d+))|(90(\.0+)?)$/'],
```
Comment (volosinpaul) [inline app/Http/Requests/Backend/Parking/UpdateParkingRequest.php]: 'The longitude format is invalid.' response on inputting valid value for longitude. The regex for longitude needs to be updated.
Latitudes range from -90 to 90.
Longitudes range from -180 to 180.

File: resources/views/backend/auth/user/includes/header-buttons.blade.php [hunk 4]
```diff
@@ -1,3 +1,3 @@
-<div class="btn-toolbar float-right" role="toolbar" aria-label="@lang('labels.general.toolbar_btn_groups')">
-    <a href="{{ route('admin.auth.user.create') }}" class="btn btn-success ml-1" data-toggle="tooltip" title="@lang('labels.general.create_new')"><i class="fas fa-plus-circle"></i></a>
```
Comment (volosinpaul) [inline resources/views/backend/auth/user/includes/header-buttons.blade.php]: User Management - Create User default functionality broken, as the link refers to admin.parking.create

File: app/Http/Requests/Backend/Parking/StoreParkingRequest.php (hunk 2 above)
Comment (maloun96) [reply app/Http/Requests/Backend/Parking/StoreParkingRequest.php]: let's skip this issues and let's focus on other tasks.

File: resources/views/backend/parking/index.blade.php [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+    <div class="card-body">
+        <div class="row">
+            <div class="col-sm-5">
+                <h4 class="card-title mb-0">
+                    Parking
+                </h4>
+            </div><!--col-->
+
+            <div class="col-sm-7">
+                @include('backend.auth.user.includes.header-buttons')
```
Comment (volosinpaul) [inline resources/views/backend/parking/index.blade.php]: Replace 'backend.auth.user.includes.header-buttons' with 'backend.parking.includes.header-buttons'.

File: resources/views/backend/parking/show.blade.php [hunk 6]
```diff
@@ -0,0 +1,52 @@
+@extends('backend.layouts.app')
+
+@section('title', __('labels.backend.access.users.management') . ' | ' . __('labels.backend.access.users.view'))
+
+@section('breadcrumb-links')
+    @include('backend.auth.user.includes.breadcrumb-links')
```
Comment (volosinpaul) [inline resources/views/backend/parking/show.blade.php]: Replace 'backend.auth.user.includes.breadcrumb-links' with 'backend.parking.includes.breadcrumb-links'.

File: database/seeds/AuthTableSeeder.php [hunk 7]
```diff
@@ -36,6 +36,7 @@ public function run()
         $this->call(UserTableSeeder::class);
         $this->call(PermissionRoleTableSeeder::class);
         $this->call(UserRoleTableSeeder::class);
+        $this->call(ParkingTableSeeder::class);
```
Comment (volosinpaul) [inline database/seeds/AuthTableSeeder.php]: Just discovered that on running php artisan db:seed ,the ParkingTableSeeder Class isn't recognised, maybe it has to be moved from database/seeds/ to database/seeds/Auth/

**Pull request merged**